### DISCUSSION TITLE
How to match a specific bloc of multiple lines?

### DISCUSSION THREAD

**Discussion by vctls** [krasa/GrepConsole#258] [Q&A] [answered]
I wish to match all lines in the following block and stop just before the last line that says `Exception trace`:
```
In PDOException.php line 20:
                                           
  [Doctrine\DBAL\Driver\PDO\PDOException]  
  There is no active transaction           
                                           

Exception trace:
```
I've tried multiple expressions that work elsewhere but not in the plugin.  
For example: https://regexr.com/79kn5

**Comment by krasa** [comment, marked as answer]
The text is matched by lines: 
![image](https://user-images.githubusercontent.com/1160875/223190439-d3842741-9ac8-42a2-8317-294885567161.png)

**Reply by vctls** [reply]
Oh, OK. This works, thanks a lot!
So "multiline" just means it will affect everything from the matching line, and then I _have to_ set a second expression to close the block?

**Reply by krasa** [reply]
yes
